### PR TITLE
Add first version of code structure and test case for src

### PR DESCRIPTION
**Pull request by cclien0725**
I have added the node.js first version of code structure, it used Expressjs with ES6 feature to implemented RESTful API, there are two available APIs(/api/singers, /api/albums) depend on doc/{singer.json, album.json}, the response data is hard code for temporary. The future work will add db module or ORM library. I have also used Mocha to write unit test for our APIs.

### CODE REVIEW

**Comment by rockers7414** [comment]
src/README.md
I think these instructions should be written in the main README.md and record what plug-in we used.
By the way, the readme also have to recode the pre-requirements or something, likes note.

File: src/api/albums.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Router } from 'express';
+import * as ModelAlbums from 'models/albums';
+
+let router = Router();
+
+router.get('/', (req, res) => {
+  ModelAlbums
+    .getAll()
+    .then(data => res.json(data));
+});
```
Comment (rockers7414) [inline src/api/albums.js]: I think res should be resp, it's more clear to know this is the response, not the resource that we use in the service.

File: src/package.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "zeus",
+  "version": "0.0.1",
+  "description": "",
```
Comment (rockers7414) [inline src/package.json]: The version should be 1.0.X. (major.minor.patch).
We define the major and minor version, and the patch should be incremented by CI server. Please change to 1.0.0.

File: src/package.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rockers7414/Zeus.git"
+  },
+  "author": {
+    "name": "RCK",
+    "email": "rockers7414@icloud.com",
+    "url": "https://github.com/rockers7414"
+  },
```
Comment (rockers7414) [inline src/package.json]: Please change the author to contributors, and record your info. Thanks.

**Comment by rockers7414** [comment]
I think this model is good for now. I'll merge this pull request after you revise the code.

File: src/api/albums.js (hunk 1 above)
Comment (cclien0725) [inline src/api/albums.js]: But, the most people who used `express` usually declare `res` to stand for response. http://expressjs.com/en/api.html#app

File: src/package.json (hunk 2 above)
Comment (cclien0725) [inline src/package.json]: Ok, i will revise it on next commit.

File: src/package.json (hunk 3 above)
Comment (cclien0725) [inline src/package.json]: According to the [npm document](The "author" is one person. "contributors" is an array of people. A "person" is an object with a "name" field and optionally "url" and "email"):
```The "author" is one person. "contributors" is an array of people. A "person" is an object with a "name" field and optionally "url" and "email".```
So, did you mean that to discard the author key/value, and add your information into contributors?

Comment (rockers7414) [inline src/package.json]: http://browsenpm.org/package.json
Append your info into the list. Thanks.

File: src/api/albums.js (hunk 1 above)
Comment (rockers7414) [inline src/api/albums.js]: Okay. Just leave it with res.

File: src/package.json (hunk 3 above)
Comment (rockers7414) [inline src/package.json]: The link is broken.
Please refer this to do:

`
"author": "Mark Pundsack",
"contributors": [
  "Zeke Sikelianos <zeke@sikelianos.com> (http://zeke.sikelianos.com)"
],
`

Comment (rockers7414) [inline src/package.json]: Sorry, I didn't see you already done. 👍

Comment (cclien0725) [inline src/package.json]: So, the `author` and `contributors` is fine for current commit version? Don't revise anything right?

File: src/api/albums.js (hunk 1 above)
Comment (cclien0725) [inline src/api/albums.js]: Ok, done.

File: src/package.json (hunk 3 above)
Comment (idl-shaohua) [inline src/package.json]: Yes.

Comment (cclien0725) [inline src/package.json]: Okay, i got it.

File: src/README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+`$ npm install`
+
+`$ npm start`
+
+### Development
+
+`$ npm run dev`
+
+`$ npm test`
+
```
Comment (cclien0725) [inline src/README.md]: Do the content of `src/README.md` have to add to the root of project  README.md?

**Comment by rockers7414** [comment]
Great job.

**Pull request merged**